GooglePay/src/main/java/com/braintreepayments/api/GooglePayLauncher.java

```diff
@@ -49,7 +49,7 @@ public GooglePayLauncher(@NonNull ComponentActivity activity,
     GooglePayLauncher(ActivityResultRegistry registry, LifecycleOwner lifecycleOwner,
                       GooglePayLauncherCallback callback) {
         activityLauncher = registry.register(GOOGLE_PAY_RESULT, lifecycleOwner,
-                new GooglePayActivityResultContract(), callback::onResult);
+                new GooglePayActivityResultContract(), callback::onGooglePayLauncherResult);
```

Comment:
It looks like this one is public - do we need to call out this change in the CHANGELOG?

Reply:
Good catch! Just pushed an update to the changelog.